assets/js/state/softwareUpdates.js

```diff
@@ -44,18 +45,51 @@ export const softwareUpdatesSlice = createSlice({
         [hostID]: { ...initialHostState, errors },
       };
     },
+    setPatchesForPackages: (state, { payload: { hostID, patches } }) => {
+      if (!state.softwareUpdates[hostID].upgradable_packages) {
+        return state;
+      }
+
+      const packages = get(state, [
+        'softwareUpdates',
+        hostID,
+        'upgradable_packages',
+      ]);
+
+      state.softwareUpdates[hostID].upgradable_packages = packages.map(
+        (currentPackage) => {
+          const { to_package_id: packageID } = currentPackage;
+
+          const packageInfo = find(patches, { package_id: packageID });
+          const packagePatches = get(packageInfo, 'patches', []);
+
+          return {
+            ...currentPackage,
+            patches: packagePatches,
+          };
+        }
+      );
+
+      return state;
```

Comment:
Do we actually need to `return state` in this function?

Reply:
ESLint yelled at me if I didn't :smile: Maybe I could architect this piece of code better, I'm really open to suggestions

Reply:
Oh hmm that's weird as pretty much all the other reducer functions in our codebase don't return anything :thinking: is that a newly added rule or something?